src/components/PDFLink/Document/index.tsx

```diff
@@ -84,7 +82,11 @@ export function createDocument({
         </View>
         <Text
           style={styles.pageNumber}
-          render={({ pageNumber, totalPages }: PageMeta) => `${pageNumber} / ${totalPages}`}
+          render={({ pageNumber, totalPages }: PageMeta) => {
+            setPageCount(totalPages !== undefined ? totalPages : 0);
```

Comment:
```suggestion
            setPageCount(totalPages || 0);
```

Reply:
Also you should move this out of the return statement into function body of `createDocument()`